### ISSUE TITLE
VLC 3.x sends error messages to stdout when starting a song

### ISSUE THREAD

**Issue by ValentijnvdBeek** [and3rson/clay#19]
Clay version: 0.7.2 
VLC version: 3.0.0

After updating VLC it will send an "[00007f15000b78e0] prefetch stream error: unimplemented query (264) in control" every time you start playing a song. This interrupts the output of Clay making the program uncontrollable and, hence, unusable.

**Comment by and3rson** [comment]
Yeah, VLC is sometimes pretty rude in terms of logging. Does this happen due to some certain conditions?
